packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -586,7 +595,7 @@ export class ScrollableTabBar extends TabBar<Widget> {
     protected scrollBar?: PerfectScrollbar;
 
     private scrollBarFactory: () => PerfectScrollbar;
-    private pendingReveal?: Promise<void>;
+    protected pendingReveal?: Promise<void>;
```

Comment:
Minor: I'm wondering why we have any `private` fields at all here. Usually, we just have everything `protected`, especially in the `core` package.

Reply:
Good point. I changed the visibility of private fields in `tab-bars.ts` class to protected.